Integrations/Gmail/Gmail.py

```diff
@@ -1028,22 +1333,361 @@ def remove_filter(user_id, _id):
+    inlineAttachments = []  # type: list
+
+    if htmlBody is not None:
+        htmlBody, htmlAttachments = handle_html(htmlBody)
+        msg = MIMEText(htmlBody, 'html', 'utf-8')
+        message.attach(msg)
+        if attach_cid is not None and len(attach_cid) > 0:
+            inlineAttachments = collect_inline_attachments(attach_cid)
+
+    else:
+        # if not html body, cannot attach cids in message
+        transientFileCID = None
+
+    attachments = collect_attachments(entry_ids, file_names)
+    manual_attachments = collect_manual_attachments()
+    transientAttachments = transient_attachments(transientFile, transientFileContent, transientFileCID)
+
+    attachments = attachments + htmlAttachments + transientAttachments + inlineAttachments + manual_attachments
+    attachment_handler(message, attachments)
+
+    encoded_message = base64.urlsafe_b64encode(message.as_string())
+    command_args = {
+        'userId': emailfrom,
+        'body': {
+            'raw': encoded_message,
+        }
+    }
+    service = get_service('gmail', 'v1', additional_scopes=['https://www.googleapis.com/auth/gmail.compose',
+                                                            'https://www.googleapis.com/auth/gmail.send'])
+    result = service.users().messages().send(**command_args).execute()
+    return result
+
+
+def send_mail_command():
+    args = demisto.args()
+    emailto = argToList(args.get('to'))
+    emailfrom = args.get('from')
+    body = args.get('body')
+    subject = args.get('subject')
+    entry_ids = argToList(args.get('attachIDs'))
+    cc = argToList(args.get('cc'))
+    bcc = argToList(args.get('bcc'))
+    htmlBody = args.get('htmlBody')
+    replyTo = args.get('replyTo')
+    file_names = argToList(args.get('attachNames'))
+    attchCID = argToList(args.get('attachCIDs'))
+    transientFile = argToList(args.get('transientFile'))
+    transientFileContent = argToList(args.get('transientFileContent'))
+    transientFileCID = argToList(args.get('transientFileCID'))
+    additional_headers = argToList(args.get('additionalHeader'))
+    template_param = args.get('templateParams')
+
+    if emailfrom is None:
+        emailfrom = ADMIN_EMAIL
+
+    result = send_mail(emailto, emailfrom, subject, body, entry_ids, cc, bcc, htmlBody,
+                       replyTo, file_names, attchCID, transientFile, transientFileContent,
+                       transientFileCID, additional_headers, template_param)
+    return sent_mail_to_entry('Email sent:', [result], emailto, emailfrom, cc, bcc, htmlBody, body, subject)
+
+
+'''FETCH INCIDENTS'''
+
+
 def fetch_incidents():
     params = demisto.params()
     user_key = params.get('queryUserKey')
     user_key = user_key if user_key else ADMIN_EMAIL
     query = '' if params['query'] is None else params['query']
     last_run = demisto.getLastRun()
-    last_fetch = last_run.get('time')
-
-    # handle first time fetch
+    last_fetch = last_run.get('gmt_time')
```

Comment:
this will cause customer duplicate incidents since after the update it will not find the `gmt_time` as the last run contains only the `time` value.

@kirbles19 
This MUST be documented properly in the RN

Reply:
I had a look at the RN after the approval and issue is not mentioned - @kirbles19 @richardbluestone - if you can please take another look and add this important clarification.
https://github.com/demisto/content/blob/gmail_add_commands/Integrations/Gmail/CHANGELOG.md